crucible-cli/src/Lang/Crucible/CLI.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE DataKinds #-}
+ {-# LANGUAGE DataKinds #-}
```

Comment:
This whitespace change was introduced here, only to be reverted in a later commit in the PR. Consider fixing up this commit to just not make this whitespace change in the first place?